frontend/src/app/containers/ExperimentPage.js

```diff
@@ -157,12 +157,13 @@ export class ExperimentDetail extends React.Component {
 
   renderLocaleWarning() {
     const { experiment, locale } = this.props;
-    if (experiment.locales && locale && !experiment.locales.includes(locale)) {
+    if (locale && ((experiment.locales && !experiment.locales.includes(locale)) || (experiment.locale_blocklist && experiment.locale_blocklist.includes(locale)))) {
```

Comment:
@ckprice Is there a better place we could put this filter code? Maybe reuse the filters in `src/app/selectors/experiment.js`?